liblangutil/Scanner.cpp

```diff
@@ -789,13 +802,22 @@ Token Scanner::scanString()
 				return setError(ScannerError::IllegalEscapeSequence);
 		}
 		else
+		{
+			// Report error on non-printable characters in string literals, however
+			// allow anything for unicode string literals, because their validity will
+			// be verified later (in the syntax checker).
+			//
+			// NOTE: remove dependency on locale and thus not use isprint() here
```

Comment:
No locale here.

Reply:
I meant to describe why we have our own range not using `isprint`.

Reply:
Changed.